enterprise/frontend/src/embedding-sdk/store/reducer.ts

```diff
@@ -113,31 +85,31 @@ export const sdk = createReducer(initialState, builder => {
     token: { ...state.token, loading: true },
```

Comment:
@npretto I tried to stack this PR on top of the Mantine V7 PR, and the SDK build still fails because of this `refreshTokenAsync.pending` case. Could you change this to use the Immer syntax?